src/content/docs/paper/dev/api/dialogs.mdx

```diff
@@ -0,0 +1,366 @@
+
+   <Video src={InputTextMp4} />
+
+- [`DialogInput.numberRange`](jd:paper:io.papermc.paper.registry.data.dialog.input.DialogInput#numberRange(java.lang.String,net.kyori.adventure.text.Component,float,float))
+
+   A slider for number input
+
+   <Video src={InputNumberRange}/>
+
+
+### Dialog type
+The [`DialogTypes`](jd:paper:io.papermc.paper.registry.data.dialog.type.DialogType) interface defines a few static
+methods for the various dialog types. The following types exist:
+
+| Type             | Method                                                                                                                                                                                                                                              | Description                                            |
+|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------|
+| notice           | [`notice()`](jd:paper:io.papermc.paper.registry.data.dialog.type.DialogType#notice()) or [`notice(ActionButton button)`](jd:paper:io.papermc.paper.registry.data.dialog.type.DialogType#notice(io.papermc.paper.registry.data.dialog.ActionButton)) | A simple dialog with just one button                   |
+| confirmation     | [`confirmation(ActionButton yesButton, ActionButton noButton)`](jd:paper:io.papermc.paper.registry.data.dialog.type.DialogType#confirmation(io.papermc.paper.registry.data.dialog.ActionButton,io.papermc.paper.registry.data.dialog.ActionButton)) | A dialog with a yes and no button                      |
+| dialog list      | [`dialogList(RegistrySet dialogs)`](jd:paper:io.papermc.paper.registry.data.dialog.type.DialogType#dialogList(io.papermc.paper.registry.set.RegistrySet))                                                                                           | A dialog for opening the specified, registered dialogs |
+| multiple actions | [`multiAction(List<ActionButton> actions)`](jd:paper:io.papermc.paper.registry.data.dialog.type.DialogType#multiAction(java.util.List))                                                                                                             | A dialog for displaying multiple buttons               |
+| server links     | [`serverLinks(ActionButton exitAction, int columns, int buttonWidth)`](jd:paper:io.papermc.paper.registry.data.dialog.type.DialogType#serverLinks(io.papermc.paper.registry.data.dialog.ActionButton,int,int))                                      | A server links dialog                                  |
+
+The type primarily influences the bottom part of the dialog.
+
+## Registering dialogs in the registry
+If you want dialogs to be registered in the dialogs registry, you have register them inside a registry modification lifecycle
```

Comment:
Should we add a remark reminding about needing to do this for the dialogList type?